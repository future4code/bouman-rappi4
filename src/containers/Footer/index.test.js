import React from 'react'
import renderer from "react-test-renderer";
import { Footer } from './index';

describe ('teste componente Footer', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Footer></Footer>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})