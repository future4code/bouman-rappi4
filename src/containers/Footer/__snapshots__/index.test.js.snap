// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste componente Footer renders correctly 1`] = `
<div
  className="MuiBottomNavigation-root-1 sc-bdVaJa blZzOz"
>
  <button
    className="MuiButtonBase-root-7 MuiBottomNavigationAction-root-2 MuiBottomNavigationAction-iconOnly-4"
    onBlur={[Function]}
    onClick={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiBottomNavigationAction-wrapper-5"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-10 undefined"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 5.69l5 4.5V18h-2v-6H9v6H7v-7.81l5-4.5M12 3L2 12h3v8h6v-6h2v6h6v-8h3L12 3z"
        />
      </svg>
      <span
        className="MuiBottomNavigationAction-label-6 MuiBottomNavigationAction-iconOnly-4"
      />
    </span>
    <span
      className="MuiTouchRipple-root-19"
    />
  </button>
  <button
    className="MuiButtonBase-root-7 MuiBottomNavigationAction-root-2 MuiBottomNavigationAction-iconOnly-4"
    onBlur={[Function]}
    onClick={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiBottomNavigationAction-wrapper-5"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-10 undefined"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.37-.66-.11-1.48-.87-1.48H5.21l-.94-2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
        />
      </svg>
      <span
        className="MuiBottomNavigationAction-label-6 MuiBottomNavigationAction-iconOnly-4"
      />
    </span>
    <span
      className="MuiTouchRipple-root-19"
    />
  </button>
  <button
    className="MuiButtonBase-root-7 MuiBottomNavigationAction-root-2 MuiBottomNavigationAction-iconOnly-4"
    onBlur={[Function]}
    onClick={[Function]}
    onContextMenu={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiBottomNavigationAction-wrapper-5"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-10 undefined"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 5.9c1.16 0 2.1.94 2.1 2.1s-.94 2.1-2.1 2.1S9.9 9.16 9.9 8s.94-2.1 2.1-2.1m0 9c2.97 0 6.1 1.46 6.1 2.1v1.1H5.9V17c0-.64 3.13-2.1 6.1-2.1M12 4C9.79 4 8 5.79 8 8s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm0 9c-2.67 0-8 1.34-8 4v3h16v-3c0-2.66-5.33-4-8-4z"
        />
      </svg>
      <span
        className="MuiBottomNavigationAction-label-6 MuiBottomNavigationAction-iconOnly-4"
      />
    </span>
    <span
      className="MuiTouchRipple-root-19"
    />
  </button>
</div>
`;
